### PR TITLE
Add docker tab in the website

### PR DESCRIPTION
**Pull request by haimasree**
Looks something like this

image

Ofcourse we must move the docker containers to kipoi dockerhub repo. As soon as you send me the  link and the credentials  I will retag the containers and update this PR. Until then this PR need not be merged. I  just thought I will get an opinion if you are happy with the design or the information contained in it. We can discuss this on Monday too.

### CODE REVIEW

File: app/models/code_snippets.py [hunk 1]
```diff
@@ -40,6 +40,94 @@ def get_group_name(model_name, source):
...
+        docker_image_name = ""
+    ctx["docker_image_name"] = docker_image_name
+    print("docker image name {}".format(docker_image_name))
+    return [("Get the docker image", """docker pull {docker_image_name}""".format(**ctx)),
+            ("Get the activated conda environment inside the container",
+             """docker run -it {docker_image_name}""".format(**ctx)
+             ),
+        ("Test the model", "docker run {docker_image_name} kipoi test {model_name} --source={source}".format(**ctx)),
+        ("Make prediction for custom files directly", """mkdir -p <absolute path to your data dir>/output \\
+docker run -v <absolute path to your data dir>:/app/ {docker_image_name} \\
```
Comment (Avsecz) [inline app/models/code_snippets.py:120]: Replace all instances `<absolute path to your data dir>` with a concrete example directory, say `PWD/kipoi-example`. That way, the user can just run this command and then adjust the directory later. Also add a note that this path can be adjusted to an arbitrary path. Here is one eample: 

```
# Create an example directory containing the data.
mkdir -p $PWD/kipoi-example
# You can replace $PWD/kipoi-example with a different absolute path containing the data.
docker run $PWD/kipoi-example:/app/ ...
```

File: app/models/code_snippets.py [hunk 2]
```diff
@@ -40,6 +40,94 @@ def get_group_name(model_name, source):
...
+kipoi get-example {model_name} -o /app/{output_dir} \\
+docker run -v <absolute path to your data dir>:/app/ {docker_image_name} \\
+kipoi predict {model_name} \\
+--dataloader_args='{example_kwargs}' \\
+-o '/app/output/{model_name_no_slash}.example_pred.tsv'
```
Comment (Avsecz) [inline app/models/code_snippets.py:125]: I'm getting an error here `ERROR [kipoi.cli.main] Directory /app/output doesn't exist``

**Review by Avsecz — CHANGES_REQUESTED** [review]

File: app/models/code_snippets.py [hunk 3]
```diff
@@ -40,6 +40,94 @@ def get_group_name(model_name, source):
...
+    """
+    Generate the snippet for the docker containers  
+
+    Args:
+        model_name (str): Name of the model the description being generated for
+        source (str, optional): Where the model is residing. Defaults to "kipoi".
+    """
+    # TODO: read from kipoi-donctainer repo directly. Need an SSH key
+    model_group_to_image_dict = {
+    "DeepCpG_DNA": "haimasree/kipoi-docker:sharedpy3keras1.2",
```
[the remark this answers is not in the document]
Comment (haimasree) [reply app/models/code_snippets.py]: Fixed. Can you verify please

File: app/models/code_snippets.py (hunk 1 above)
Comment (haimasree) [reply app/models/code_snippets.py]: Great idea! Done.

File: app/models/code_snippets.py (hunk 2 above)
Comment (haimasree) [reply app/models/code_snippets.py]: Oh no. Can you please check again?

**Comment by Avsecz** [comment]
Works like a charm. Great job!

**Pull request merged**